src/components/App/App.js

```diff
@@ -20,23 +20,28 @@ const client = new ApolloClient({
 export const App = () => {
   const [user, setUser] = useState('');
   const [interviews, setInterviews] = useState([]);
+  const [questions, setQuestions] = useState([]);
 
   const userInfo = useMemo(() => ({ user, setUser }), [user, setUser]);
   const fetchedInterviews = useMemo(() => ({ interviews, setInterviews }), [interviews, setInterviews]);
+  const fetchedQuestions = useMemo(() => ({ questions, setQuestions }), [questions, setQuestions]);
 
   return (
     <ApolloProvider client={client}>
       <main className='main'>
-        <InterviewsContext.Provider value={fetchedInterviews}>
-          <UserContext.Provider value={userInfo}>
-            <NavBar />
-            <Route exact path='/'><HomePage /></Route>
-            <Route exact path='/login' render={() => user ? (<Redirect to='/dashboard'/>) : <LoginForm />}/>
-            <Route exact path='/dashboard'><Dashboard /></Route>
-            <Route exact path='/student-interviews'><StudentInterviewContainer /></Route>
-            <Route exact path='/interview'><Interview/></Route>
-          </UserContext.Provider>
-        </InterviewsContext.Provider>
+        <QuestionsContext.Provider value={fetchedQuestions}>
+          <InterviewsContext.Provider value={fetchedInterviews}>
+            <UserContext.Provider value={userInfo}>
```

Comment:
Is there a way to group these under a single component? Not saying you should, just curious.

Reply:
Not sure, although I don't believe so. Each context needs its own default state so I don't know if there is a way to group them in a single component. I do like the idea of potentially storing all of the user's info (notes, questions, etc) as a single object in a single context.

Reply:
Yeah no worries I was just thinking that we'd run into some issues once we implemented the queries and realised we didn't have a way to display multiple notes.